### PR TITLE
feat: update step-registry to use upstream tests

### PR DESCRIPTION
**Pull request by vprashar2929**
This PR updates the step-registry to use upstream e2e test's to run against OpenShift.

### CODE REVIEW

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-operator-v1alpha1-kepler-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-operator-v1alpha1-kepler-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-operator-v1alpha1-kepler-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-operator-v1alpha1-kepler-operator-e2e

File: ci-operator/step-registry/openshift-power-monitoring/tests/kepler/openshift-power-monitoring-tests-kepler-commands.sh [hunk 1]
```diff
@@ -6,9 +6,9 @@ must_gather() {
     oc get pods -n openshift-operators
     oc describe deployment kepler-operator-controller-manager -n openshift-operators
     oc logs -n openshift-operators deployments/kepler-operator-controller-manager
-    oc get pods -n openshift-kepler-operator
-    oc describe daemonsets/kepler-exporter-ds -n openshift-kepler-operator
-    oc logs -n daemonsets/kepler-exporter-ds -n openshift-kepler-operator
+    # oc get pods -n openshift-kepler-operator
+    # oc describe daemonsets/kepler-exporter-ds -n openshift-kepler-operator
+    # oc logs -n daemonsets/kepler-exporter-ds -n openshift-kepler-operator
```
Comment (sthaha) [inline ci-operator/step-registry/openshift-power-monitoring/tests/kepler/openshift-power-monitoring-tests-kepler-commands.sh:11]: Is this intentional? Could you please add a note why this is commented?

File: ci-operator/step-registry/openshift-power-monitoring/tests/kepler/openshift-power-monitoring-tests-kepler-commands.sh [hunk 2]
```diff
@@ -25,18 +25,23 @@ validate_kepler() {
...
+    # url=http://$(oc get route kepler-metrics-route -n openshift-kepler-operator -o jsonpath='{.spec.host}')/metrics
+    # [[ $(curl "$url" | grep -c kepler_) -gt 0 ]] || {
+    #     echo "Kepler validation failed"
+    #     must_gather
+    #     return 1
+    # }
+    # echo "Kepler validation successful"
+    ls -ltr /tmp/
+    ./tests/run-e2e.sh --no-deploy --no-builds --ns openshift-operators || {
         must_gather
```
Comment (sthaha) [inline ci-operator/step-registry/openshift-power-monitoring/tests/kepler/openshift-power-monitoring-tests-kepler-commands.sh:43]:
```suggestion
        must_gather || true
```

File: ci-operator/step-registry/openshift-power-monitoring/tests/kepler/openshift-power-monitoring-tests-kepler-commands.sh (hunk 1 above)
Comment (vprashar2929) [reply ci-operator/step-registry/openshift-power-monitoring/tests/kepler/openshift-power-monitoring-tests-kepler-commands.sh:11]: Yes, It's not ready to merge. I am still testing some bits🙂

**Comment by sthaha** [comment]
I see, could you please add a WIP label or /hold if the pr isn't ready for merge 👼 ?

**Pull request closed without merging**